client/src/components/TagField.js

```diff
@@ -124,7 +175,7 @@ class TagField extends Component {
     }
 
     // if this is a single select then we just need the first value
-    if (!passThroughAttributes.multi && passThroughAttributes.value) {
+    if (!multi && passThroughAttributes.value) {
```

Comment:
This isn't a passthrough anymore - missed this the first attempt when I pulled multi out as a distinct variable.